### PR TITLE
Create HTTP, Texto.

### PR DESCRIPTION
**Pull request by Juanbh891063**
### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: HTTP, Texto. [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+</reqiered, tex,h1>;
+</lgm link>
+</video<h1>
+<h1> primera imagen en html</h1>
+
+
+Imagen formato </b>:
+</b>
+<video Src= “http.//link
+<div>
```
Comment (Marksmith1987) [inline HTTP, Texto.:11]: HTTP, Texto.an andrew I think that is posted indiscriminately wgj

**Review by Marksmith1987 — APPROVED** [review]

File: HTTP, Texto. (hunk 1 above)
Comment (Dziuniekhm) [reply HTTP, Texto.:11]: > HTTP, Texto.an andrew I think that is posted indiscriminately wgj

**Review by Dziuniekhm — APPROVED** [review]
https://github.com/github/docs/pull/10302/commits/d688af0813f2ad6bce521761f566e622cec7904b#r715985480

File: HTTP, Texto. (hunk 1 above)
Comment (Dziuniekhm) [reply HTTP, Texto.:11]: > HTTP, Texto.an andrew I think that is posted indiscriminately wgj

**Review by Dziuniekhm — APPROVED** [review]

**Comment by Finezze2020** [comment]
> ### Why:
> Closes [issue link]
> 
> ### What's being changed:
> ### Check off the following:
> * [x]  I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

**Review by Finezze2020 — APPROVED** [review]
#10329 #6326

**Review by Dstiers1191 — APPROVED** [review]

**Review by Finezze2020 — APPROVED** [review]

**Pull request closed without merging**